modules/extension/ysld/src/test/java/org/geotools/ysld/parse/UtilTest.java

```diff
@@ -29,12 +27,12 @@
     public void testColor() {
         Factory factory = new Factory();
 
-        assertEquals(new Color(0xFF0000), ((Literal) Util.color("FF0000", factory)).getValue());
-        assertEquals(new Color(0x00FF00), ((Literal) Util.color("00ff00", factory)).getValue());
-        assertEquals(new Color(0x0000FF), ((Literal) Util.color("#0000FF", factory)).getValue());
-        assertEquals(new Color(0xFFFF00), ((Literal) Util.color("0xFFFF00", factory)).getValue());
-        assertEquals(new Color(0x00FFFF),
+        assertEquals("#FF0000", ((Literal) Util.color("FF0000", factory)).getValue());
```

Comment:
We need to make sure that litearls with color objects are still supported, please do not remove the above tests, but you can add to them :)